pkg/client/userd/trafficmgr/traffic_manager.go

```diff
@@ -385,13 +386,16 @@ func connectMgr(
 	}
 
 	dlog.Debug(ctx, "checking that traffic-manager exists")
-	existing, _, isManagerErr := helm.IsTrafficManager(ctx, cluster.ConfigFlags, cluster.GetManagerNamespace())
-	if isManagerErr == nil && existing == nil {
-		return nil, errcat.User.New("traffic manager not found, if it is not installed, please run 'telepresence helm install'")
-	}
-
-	if isManagerErr != nil {
-		dlog.Infof(ctx, "unable to look for existing helm release: %v. Assuming it's there and continuing...", isManagerErr)
+	coreV1 := k8sapi.GetK8sInterface(ctx).CoreV1()
```

Comment:
So this leaves us with two ways of detecting the traffic-manager install? One in `helm.IsTrafficManager` and one right here. I guess what I wonder is, why isn't this code part of `helm.IsTrafficManager`?

Reply:
My logic was that `helm.IsTrafficManager` checks the helm release metadata, not the presence of a kubernetes service. I think what we should actually do is make it private, since now it's only used within the helm package. I can move this logic into some `trafficmgr.ServiceInstalled` or similar

Reply:
Yes,that sounds like a good idea.